### PR TITLE
Document our Guilds.

### PR DESCRIPTION
**Pull request by grundleborg**
#### Summary

Document our Engineering Guilds.

### CODE REVIEW

**Review by gabrieljackson — APPROVED** [review]
Looks good on the cloud guild side of things!

File: site/content/contribute/getting-started/guilds.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Engineering Guilds"
+date: 2021-01-25T16:50:35+00:00
+weight: 50
+subsection: Getting Started
+---
+
+Our Engineering Guilds are a forum for sharing knowledge, discussing and agreeing plans and disseminating information related to a particular subject or technical topic. Many (although not all) Engineering Guilds are affiliated to a particular "platform" that forms a part of Mattermost, and are therefore associated with a particular Platform Team. However, the participants in each Guild span across many engineering teams at Mattermost in order to get people together to share knowledge and disseminate decisions across the Engineering organisation.
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:8]:
```suggestion
Our Engineering Guilds are a forum for sharing knowledge, discussing and agreeing on plans, and disseminating information related to a particular subject or technical topic. Many (although not all) Engineering Guilds are affiliated to a particular "platform" that forms a part of Mattermost, and are therefore associated with a particular Platform Team. However, the participants in each Guild span many engineering teams at Mattermost in order to get people together to share knowledge and disseminate decisions across the Engineering organization.
```

File: site/content/contribute/getting-started/guilds.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+The main activity of Guilds is a regular meeting. These meetings are public, and anyone within Mattermost Inc, or the wider community are welcome to attend. Details are provided below on each of the guilds, including how to participate.
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:10]:
```suggestion
The main activity of Guilds is a regular meeting. These meetings are public, and anyone within Mattermost and the wider community is welcome to attend. Details are provided below on each of the Guilds, including how to participate.
```

File: site/content/contribute/getting-started/guilds.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+The Mobile Guild covers all areas related to mobile application development and React Native.
+
+- **Guild Lead:** Elias Nahum
+- **Guild Channel:** [~]()
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
+
+## Server/Backend Guild
+
+The server guild covers all areas related to our server codebase and general Go development at Mattermost.
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:30]:
```suggestion
The Server/Backend Guild covers all areas related to our server codebase and general Go development at Mattermost.
```

**Review by mgdelacroix — APPROVED** [review]

**Comment by hanzei** [comment]
@grundleborg Would this also be the place to document the `Community Guild` despite it isn't a "Engineering guild"? Do you see value in having it alongside the other guilds?

cc @jasonblais

**Comment by grundleborg** [comment]
@hanzei yes good idea, I think it makes sense to include it here. Would you be able to add a suggested edit to this PR with the details in the same format as the other Guilds?

File: site/content/contribute/getting-started/guilds.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- **Guild Channel:** [~Developers: Cloud](https://community-daily.mattermost.com/core/channels/cloud)
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
+
+## QA Guild
+
+The QA Guild covers all areas related to the Quality Assurance and Automated Testing infrastructure at Mattermost.
+
+- **Guild Lead:** Linda Mitchell
+- **Guild Channel:** [~QA: Weekly Meetings](https://community.mattermost.com/core/channels/qa-weekly-meetings)
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
```
Comment (hanzei) [inline site/content/contribute/getting-started/guilds.md:50]:
```suggestion
- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*

## Contributor Community Guild

The Contributor Community Guild covers all areas related to the Mattermost Community like the Contributor Experience, Events and the health of the Community.

- **Guild Lead:** Jason Blais
- **Guild Channel:** [~guild-contributor-community ](https://community.mattermost.com/core/channels/guild-contributor-community)
- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*

```

cc @jasonblais

File: site/content/contribute/getting-started/guilds.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+The main activity of Guilds is a regular meeting. These meetings are public, and anyone within Mattermost and the wider community is welcome to attend. Details are provided below on each of the Guilds, including how to participate.
+
+## Web Application & Desktop Guild
+
+The Web Application & Desktop Guild covers all areas related to front-end web development and Electron desktop app development at Mattermost.
+
+- **Guild Lead:** Dean Whillier
+- **Guild Channel:** [~Developers: WebApp](https://community.mattermost.com/core/channels/webapp)
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
```
Comment (hanzei) [inline site/content/contribute/getting-started/guilds.md:18]: Should that be 
```suggestion
- **Guild Meeting Time:** *(see header of Guild Channel for meeting invite)*
```
? It wasn't obvious to me what that means.

Comment (grundleborg) [reply site/content/contribute/getting-started/guilds.md:18]: 0/5 personally. I'll let @justinegeffen advise as editor.

File: site/content/contribute/getting-started/guilds.md (hunk 4 above)
Comment (grundleborg) [reply site/content/contribute/getting-started/guilds.md:50]: Thanks!

File: site/content/contribute/getting-started/guilds.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+The QA Guild covers all areas related to the Quality Assurance and Automated Testing infrastructure at Mattermost.
+
+- **Guild Lead:** Linda Mitchell
+- **Guild Channel:** [~QA: Weekly Meetings](https://community.mattermost.com/core/channels/qa-weekly-meetings)
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
+
+## Contributor Community Guild
+
+The Contributor Community Guild covers all areas related to the Mattermost Community like the Contributor Experience, Events and the health of the Community.
```
Comment (jasonblais) [inline site/content/contribute/getting-started/guilds.md:54]:
```suggestion
The Contributor Community Guild covers all areas related to the Mattermost Contributor Community like the contributor experience, code contributions, recognition, and more.
```

Comment (jasonblais) [reply site/content/contribute/getting-started/guilds.md:54]: @hanzei what are your thoughts on the above adjustments?

File: site/content/contribute/getting-started/guilds.md (hunk 5 above)
Comment (justinegeffen) [reply site/content/contribute/getting-started/guilds.md:18]: Good suggestion, @hanzei.

File: site/content/contribute/getting-started/guilds.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+Our Engineering Guilds are a forum for sharing knowledge, discussing and agreeing on plans, and disseminating information related to a particular subject or technical topic. Many (although not all) Engineering Guilds are affiliated to a particular "platform" that forms a part of Mattermost, and are therefore associated with a particular Platform Team. However, the participants in each Guild span many engineering teams at Mattermost in order to get people together to share knowledge and disseminate decisions across the Engineering organization.
+
+The main activity of Guilds is a regular meeting. These meetings are public, and anyone within Mattermost and the wider community is welcome to attend. Details are provided below on each of the Guilds, including how to participate.
+
+## Web Application & Desktop Guild
+
+The Web Application & Desktop Guild covers all areas related to front-end web development and Electron desktop app development at Mattermost.
+
+- **Guild Lead:** Dean Whillier
+- **Guild Channel:** [~Developers: WebApp](https://community.mattermost.com/core/channels/webapp)
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:17]:
```suggestion
- **Guild channel:** [~Developers: WebApp](https://community.mattermost.com/core/channels/webapp)
```

File: site/content/contribute/getting-started/guilds.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+- **Guild Meeting Time:** *(see header of Guild Channel for meeting invite)*
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:18]:
```suggestion
- **Guild meeting time:** *(see header of Guild channel for meeting invite)*
```

File: site/content/contribute/getting-started/guilds.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+## Mobile Guild
+
+The Mobile Guild covers all areas related to mobile application development and React Native.
+
+- **Guild Lead:** Elias Nahum
+- **Guild Channel:** [~]()
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:25]:
```suggestion
- **Guild channel:** [~]()
```

File: site/content/contribute/getting-started/guilds.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:26]:
```suggestion
- **Guild meeting time:** *(see header of Guild channel for meeting invite)*
```

File: site/content/contribute/getting-started/guilds.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+- **Guild Channel:** [~QA: Weekly Meetings](https://community.mattermost.com/core/channels/qa-weekly-meetings)
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
+
+## Contributor Community Guild
+
+The Contributor Community Guild covers all areas related to the Mattermost Contributor Community like the contributor experience, code contributions, recognition, and more.
+
+- **Guild Lead:** Jason Blais
+- **Guild Channel:** [~guild-contributor-community ](https://community.mattermost.com/core/channels/guild-contributor-community)
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:58]:
```suggestion
- **Guild meeting time:** *(see header of Guild channel for meeting invite)*
```

File: site/content/contribute/getting-started/guilds.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+- **Guild Lead:** Linda Mitchell
+- **Guild Channel:** [~QA: Weekly Meetings](https://community.mattermost.com/core/channels/qa-weekly-meetings)
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
+
+## Contributor Community Guild
+
+The Contributor Community Guild covers all areas related to the Mattermost Contributor Community like the contributor experience, code contributions, recognition, and more.
+
+- **Guild Lead:** Jason Blais
+- **Guild Channel:** [~guild-contributor-community ](https://community.mattermost.com/core/channels/guild-contributor-community)
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:57]:
```suggestion
- **Guild channel:** [~guild-contributor-community ](https://community.mattermost.com/core/channels/guild-contributor-community)
```

File: site/content/contribute/getting-started/guilds.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+- **Guild Lead:** Gabe Jackson
+- **Guild Channel:** [~Developers: Cloud](https://community-daily.mattermost.com/core/channels/cloud)
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
+
+## QA Guild
+
+The QA Guild covers all areas related to the Quality Assurance and Automated Testing infrastructure at Mattermost.
+
+- **Guild Lead:** Linda Mitchell
+- **Guild Channel:** [~QA: Weekly Meetings](https://community.mattermost.com/core/channels/qa-weekly-meetings)
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:49]:
```suggestion
- **Guild channel:** [~QA: Weekly Meetings](https://community.mattermost.com/core/channels/qa-weekly-meetings)
```

File: site/content/contribute/getting-started/guilds.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:50]:
```suggestion
- **Guild meeting time:** *(see header of Guild channel for meeting invite)*
```

File: site/content/contribute/getting-started/guilds.md [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+- **Guild Lead:** Elias Nahum
+- **Guild Channel:** [~]()
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
+
+## Server/Backend Guild
+
+The Server/Backend Guild covers all areas related to our server codebase and general Go development at Mattermost.
+
+- **Guild Lead:** Miguel de la Cruz
+- **Guild Channel:** [~Developers: Server](https://community.mattermost.com/core/channels/developers-server)
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:33]:
```suggestion
- **Guild channel:** [~Developers: Server](https://community.mattermost.com/core/channels/developers-server)
```

File: site/content/contribute/getting-started/guilds.md [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:34]:
```suggestion
- **Guild meeting time:** *(see header of Guild channel for meeting invite)*
```

File: site/content/contribute/getting-started/guilds.md [hunk 17]
```diff
@@ -0,0 +1,58 @@
...
+
+## Cloud Guild
+
+The Cloud Guild covers all areas related to our cloud infrastructure and Kubernetes orchestration of Mattermost.
+
+- **Guild Lead:** Gabe Jackson
+- **Guild Channel:** [~Developers: Cloud](https://community-daily.mattermost.com/core/channels/cloud)
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:41]:
```suggestion
- **Guild channel:** [~Developers: Cloud](https://community-daily.mattermost.com/core/channels/cloud)
```

File: site/content/contribute/getting-started/guilds.md [hunk 18]
```diff
@@ -0,0 +1,58 @@
...
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/guilds.md:42]:
```suggestion
- **Guild meeting time:** *(see header of Guild channel for meeting invite)*
```

**Review by justinegeffen — APPROVED** [review]
Revised the formatting of the Guild details, but otherwise LGTM. :)

File: site/content/contribute/getting-started/guilds.md [hunk 19]
```diff
@@ -0,0 +1,58 @@
...
+
+- **Guild Lead:** Linda Mitchell
+- **Guild Channel:** [~QA: Weekly Meetings](https://community.mattermost.com/core/channels/qa-weekly-meetings)
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
+
+## Contributor Community Guild
+
+The Contributor Community Guild covers all areas related to the Mattermost Contributor Community like the contributor experience, code contributions, recognition, and more.
+
+- **Guild Lead:** Jason Blais
```
Comment (jasonblais) [inline site/content/contribute/getting-started/guilds.md:56]:
```suggestion
- **Guild Lead:** Emily Cook
```

Comment (jasonblais) [reply site/content/contribute/getting-started/guilds.md:56]: Talked with Emily, she offered to take the lead for this guild.

**Review by jasonblais — CHANGES_REQUESTED** [review]

File: site/content/contribute/getting-started/guilds.md (hunk 7 above)
Comment (deanwhillier) [inline site/content/contribute/getting-started/guilds.md:17]:
```suggestion
## Web Platform Guild

The Web Platform Guild covers all areas related to front-end web development and Electron desktop app development at Mattermost.

- **Guild Lead:** Jesse Hallam
- **Guild channel:** [~Developers: WebApp](https://community.mattermost.com/core/channels/webapp)
```

**Review by deanwhillier — CHANGES_REQUESTED** [review]
@lieut-data, I would suggest you review the Web Platform Guild section. I have proposed some changes to start.

**Review by lieut-data — APPROVED** [review]

**Review by jasonblais — APPROVED** [review]

**Review by deanwhillier — APPROVED** [review]

File: site/content/contribute/getting-started/guilds.md [hunk 20]
```diff
@@ -0,0 +1,58 @@
...
+- **Guild Lead:** Jesse Hallam
+- **Guild channel:** [~Developers: WebApp](https://community.mattermost.com/core/channels/webapp)
+- **Guild meeting time:** *(see header of Guild channel for meeting invite)*
+
+## Mobile Guild
+
+The Mobile Guild covers all areas related to mobile application development and React Native.
+
+- **Guild Lead:** Elias Nahum
+- **Guild channel:** [~]()
```
Comment (enahum) [inline site/content/contribute/getting-started/guilds.md:25]:
```suggestion
- **Guild channel:** [~Developers: Mobile](https://community.mattermost.com/core/channels/native-mobile-apps)
```

**Review by enahum — APPROVED** [review]

**Review by lindalumitchell — APPROVED** [review]

**Comment by justinegeffen** [comment]
@hanzei, this PR has an issue with github pages but everything else is building successfully. Can we merge this?

**Comment by hanzei** [comment]
@justinegeffen Yes

**Pull request merged**